From 34bed3ceb7cd2bb43e67acce97f4cc3e8bbc1c1d Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@math.unl.edu>
Date: Tue, 11 Mar 2014 09:51:17 -0500
Subject: [PATCH 1/3] use LIB_PYTHON realpath

Use GET_FILENAME_COMPONENT( ... REALPATH). PYTHON_LIBRARY as returned
by cmake, whose target is often a symlink.  Some distro packaging
reserves such library symlinks for -devel and not runtime.

REVIEW: 116719
---
 kpythonpluginfactory/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kpythonpluginfactory/CMakeLists.txt b/kpythonpluginfactory/CMakeLists.txt
index c24160e..a777dac 100644
--- a/kpythonpluginfactory/CMakeLists.txt
+++ b/kpythonpluginfactory/CMakeLists.txt
@@ -3,7 +3,8 @@
 set(kpythonpluginfactory_SRCS
     kpythonpluginfactory.cpp)
 
-GET_FILENAME_COMPONENT(LIB_PYTHON ${PYTHON_LIBRARIES} NAME)
+GET_FILENAME_COMPONENT(PYTHON_LIBRARY_REALPATH "${PYTHON_LIBRARY}" REALPATH)
+GET_FILENAME_COMPONENT(LIB_PYTHON ${PYTHON_LIBRARY_REALPATH} NAME)
 ADD_DEFINITIONS(-DLIB_PYTHON="${LIB_PYTHON}")
 ADD_DEFINITIONS(-DKDE_DEFAULT_DEBUG_AREA=15000)
 
-- 
2.9.3

